doc/_data/schemas/mc_rbdyn/Collision.json

```diff
@@ -5,8 +5,10 @@
   {
     "body1": { "type": "string", "description": "First convex body used, wildcards can be used" },
     "body2": { "type": "string", "description": "Second convex body used, wildcards can be used" },
-    "r1Joints": { "type": "array", "items": { "type": "string" }, "description": "Active joints used to avoid the collision (default: all)" },
-    "r2Joints": { "type": "array", "items": { "type": "string" }, "description": "Active joints used to avoid the collision (default: all)" },
+    "r1ActiveJoints": { "type": "array", "items": { "type": "string" }, "description": "Active joints used to avoid the collision (default: all)" },
+    "r2ActiveJoints": { "type": "array", "items": { "type": "string" }, "description": "Active joints used to avoid the collision (default: all)" },
+    "r1UnactiveJoints": { "type": "array", "items": { "type": "string" }, "description": "Joints not used to avoid the collision (default: none). Has no effect if r1ActiveJoints isn't empty." },
```

Comment:
It should be `inactive` and not `unactive`, `unactive` is a left-over from Tasks but it's incorrect.